components/org.wso2.carbon.identity.tag.manager.core/src/main/java/org/wso2/carbon/identity/tag/manager/core/constant/Constants.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.tag.manager.core.constant;
+
+/**
+ * This class contains the constants related to tag management.
+ */
+public class Constants {
```

Comment:
Shall we change this to a meaningful name. Example:- TagMgtConstants

Reply:
done